connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -475,7 +475,7 @@ private synchronized void recordSent(final ProducerRecord<byte[], byte[]> record
     public boolean commitOffsets() {
         long commitTimeoutMs = workerConfig.getLong(WorkerConfig.OFFSET_COMMIT_TIMEOUT_MS_CONFIG);
 
-        log.info("{} Committing offsets", this);
+        log.debug("{} Committing offsets", this);
```

Comment:
nit: since this is indicating entry into a method, I think it can be trace level.